source/stylesheets/refills/_modal.scss

```diff
@@ -119,34 +97,24 @@
     }
 
     h1 {
-      color: $base-font-color;
-      margin-bottom: 0.6em;
+      margin: 0 0 0.6em 0;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 0 0.6em` instead of `0 0 0.6em 0`